pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -286,6 +286,9 @@ func (c Check) validateTarget() error {
 	case CheckTypeMultiHttp:
 		// TODO(mem): checks MUST have a target, but in this case it's
 		// not true that the target must be a valid URL.
+		if strings.Contains(c.Target, "${") {
```

Comment:
why would the target of the check contain a replacement?

Reply:
I'm sure there's a better way to do this, but basically I want the URL to be validated _unless_ we detect a variable as part of the string. And if it _is_ part of the string, just bail out on validating altogether

Reply:
If you mean from a user perspective, interacting with a rest api could end up needing a variable in the url. 

- Make a request for a list of products to `https://www.russstore.com/products/list`
- Extract an individual product from the list and save it to a variable
- Make a request for the specific product at `https://www.russtore.com/products/${productId}`

Something like that